README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Personal Profile
+## Name: Xumeng Yan
+Aspect | Information
+------------ | -------------
+**Pronoun** | She/Her/Hers
+**Program** | MAPSS
+**Major** | Sociology
+**Research Interest** | *Social Stratification, Social Movements*
+
+* A couple things I enjoy:
+  * Experimenting with cooking recipes
+  * Chasing my pets
+  * Lying on the floor and silently listening to music
+
+* Favorite lyrics:
+> *If you want to make the world a better place*
+> *Take a look at yourself, and then make a change* (Michael Jackson)
+
+![Michael Jackson](Michael_Jackson_-_Man_in_the_Mirror.png)
```

Comment:
It is really cool adding images to Markdown, however in my case it didn't render. I think that checking many times that elements are correctly attached is a good practice!

Reply:
Great advice! I thought that with one space it was enough.